types/pnpapi/index.d.ts

```diff
@@ -21,21 +21,28 @@ export const VERSIONS: { std: number; [key: string]: number };
 
 export const topLevel: { name: null; reference: null };
 
-export function getPackageInformation(locator: PackageLocator): PackageInformation;
+export function getLocator(name: string, reference: string): PackageLocator;
+export function getDependencyTreeRoots(): PackageLocator[];
+export function getAllLocators(): PackageLocator[];
+export function getPackageInformation(locator: PackageLocator): PackageInformation | null;
```

Comment:
The referenced website doesn't mention this function returning `null`: https://yarnpkg.com/advanced/pnpapi#getpackageinformation